test/libsolidity/syntaxTests/nameAndTypeResolution/239_multi_variable_declaration_wildcards_fine.sol

```diff
@@ -13,7 +13,8 @@ contract C {
     }
 }
 // ----
-// Warning: (179-198): Different number of components on the left hand side (2) than on the right hand side (3).
-// Warning: (208-233): Different number of components on the left hand side (3) than on the right hand side (2).
-// Warning: (243-262): Different number of components on the left hand side (2) than on the right hand side (3).
-// Warning: (272-297): Different number of components on the left hand side (3) than on the right hand side (2).
+// TypeError: (179-198): Different number of components on the left hand side (2) than on the right hand side (3).
+// TypeError: (208-233): Different number of components on the left hand side (3) than on the right hand side (2).
+// TypeError: (243-262): Different number of components on the left hand side (2) than on the right hand side (3).
+// TypeError: (272-297): Different number of components on the left hand side (3) than on the right hand side (2).
+// TypeError: (335-350): Variable declaration statement does not declare any variables.
```

Comment:
The last two refer to weird leftover ``var`` cases that should be disallowed in a separate PR.

Reply:
The test is called "..._fine", also it mentions "wildcards", so I'm not sure if the test makes any sense at all anymore.

Reply:
Also there are other tests with "wildcards" in their name, please modify all of them (or just delete them if we already have enough tests) so that they match what we actually want to test.